### PR TITLE
Remove duplicate AUTHORS file in repo

### PR DESCRIPTION
**Pull request by stv0g**
Most of the people in the old file are anyway listed in the AUTHORS.txt file.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,10 +0,0 @@
-Sean DuBois <https://github.com/Sean-Der>
-Raphael Randschau <https://github.com/nicolai86>
-Aleksandr Razumov <ar@cydev.ru>
-Aliaksandr Valialkin <valyala@gmail.com>
```
Comment (at-wat) [inline AUTHORS:4]: Not in new AUTHORS.txt

File: AUTHORS [hunk 2]
```diff
@@ -1,10 +0,0 @@
...
-Michiel De Backker <https://github.com/backkem>
-Y.Horie <https://github.com/u5surf>
-songjiayang <https://github.com/songjiayang>
-The gortc project
-The IETF Trust
```
Comment (at-wat) [inline AUTHORS:9]: I heard this repository was originally from gortc organization.

**Review by at-wat — COMMENTED** [review]
Seems some person and organizations will be missing in the new list.
If they have contributions not in form of git commits, I think removing them is not polite.

File: AUTHORS (hunk 2 above)
Comment (stv0g) [reply AUTHORS:9]: What do you think about adding them directly to the Contributing section of the README?

I think the generate-authors script from pion/goassets is not flexible enough to handle this case..

File: AUTHORS (hunk 1 above)
Comment (stv0g) [reply AUTHORS:4]: Added to README

File: AUTHORS (hunk 2 above)
Comment (stv0g) [reply AUTHORS:9]: I added them to the README

**Comment by stv0g** [comment]
Hi @at-wat, I aligned this repo with the recent PR in the goassets repo.

File: AUTHORS.txt [hunk 3]
```diff
@@ -34,4 +34,6 @@ Yutaka Takeda <yt0916@gmail.com>
 ZHENK <chengzhenyang@gmail.com>
 
 # List of contributors not appearing in Git history
-
+The gortc project
+The IETF Trust
+Aliaksandr Valialkin <valyala@gmail.com>
```
Comment (at-wat) [inline AUTHORS.txt:39]: Generate-authors script will sort this block but not automatically pushed since this PR is from the fork.

Comment (stv0g) [reply AUTHORS.txt]: Thanks :) I fixed it in the latest version.

**Pull request merged**